packages/ava/src/ntv/ntv-types.ts

```diff
@@ -0,0 +1,22 @@
+/* export all needed types of NtvTypes */
```

Comment:
Do you want to directly name the file as `types.ts`? We can set a standard, cc @neoddish

Reply:
yes

Reply:
Fixed